src/io/github/sspanak/tt9/ime/modes/ModeABC.java

```diff
@@ -10,6 +10,7 @@ public class ModeABC extends InputMode {
 	public int getId() { return MODE_ABC; }
 
 	private boolean shouldSelectNextLetter = false;
+	private int lastKey = -1;
```

Comment:
Let's rename this for consistency with `ModePredictive`
```suggestion
	private int digit = -1;
```

Reply:
sounds reasonable